docs/configuration.md

```diff
@@ -456,6 +456,29 @@ Apart from these, the following properties are also available, and may be useful
     from JVM to Python worker for every task.
   </td>
 </tr>
+<tr>
+  <td><code>spark.sql.repl.eagerEval.enabled</code></td>
+  <td>false</td>
+  <td>
+    Enable eager evaluation or not. If true and REPL you are using supports eager evaluation,
+    dataframe will be ran automatically and html table will feedback the queries user have defined
+    (see <a href="https://issues.apache.org/jira/browse/SPARK-24215">SPARK-24215</a> for more details).
+  </td>
+</tr>
+<tr>
+  <td><code>spark.sql.repl.eagerEval.maxNumRows</code></td>
+  <td>20</td>
+  <td>
+    Default number of rows in HTML table.
```

Comment:
Shell we explain a bit more what's the HTML table here a bit more? For example, I think at least we should say it's `_repr_html_`.

Reply:
Got it, more detailed description in 7f43a8b. Please check.